### PR TITLE
PRG-87

### PR DESCRIPTION
**Pull request by felixthea**
As you can see I had a lot of fun pushing random changes.

Purpose of this pull request is to use BuildKite to kick off the tests and to push the coverage details to Code Climate.

### CODE REVIEW

**Comment by p33rs** [comment]
LGTM, although there's an open question as to whether this gets merged here or in the PRG fork.

**Comment by jayshao** [comment]
@dagifted1 your thoughts? I think we definitely want this merged here, and then would want to do a pull from origin in other repose to pickup the BK config and docker changes

**Comment by dagifted1** [comment]
Build passes in 2m4s

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+steps:
+    - command: 
+      - docker-compose -f .buildkite/docker-compose.buildkite.yml run starter-bundle 
+        npm install
+      - docker-compose -f .buildkite/docker-compose.buildkite.yml run starter-bundle 
+        npm test
+      - docker-compose -f .buildkite/docker-compose.buildkite.yml run
+        -e "CODECLIMATE_REPO_TOKEN=$CODECLIMATE_REPO_TOKEN"
+        starter-bundle
+        ./node_modules/.bin/codeclimate-test-reporter < coverage/lcov.info
```
[the remark this answers is not in the document]
Comment (jayshao) [reply .buildkite/pipeline.yml]: done

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "start": "node node_modules/.bin/webpack-dev-server --define process.env.NODE_ENV='\"development\"' --hot --inline --open --progress --colors --bail --content-base src",
     "prod": "./node_modules/.bin/webpack --define process.env.NODE_ENV='\"production\"' --progress --colors",
     "dev": "./node_modules/.bin/webpack --define process.env.NODE_ENV='\"development\"' --progress --colors",
-    "test": "./node_modules/.bin/nyc ./node_modules/.bin/ava --verbose"
+    "test": "./node_modules/.bin/nyc --reporter=lcov --reporter=html ./node_modules/.bin/ava --verbose"
```
[the remark this answers is not in the document]
Comment (jayshao) [reply package.json]: actually - I think I'm just going to fix this... -I agree, multiple reporters is nice for quick checks

**Pull request merged**